vsts-compliance.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt in the project root for license information.
+
+variables:
+  BuildConfiguration: Release
+  TeamName: vssetup
+
+trigger:
```

Comment:
Should we use a cron trigger here to ensure we run this on a schedule? Otherwise, if there are no changes we will never run the compliance build

Reply:
I've thought about using cron, but there seems to be an option to make sure to run weekly on Azure pipelines. I'm thinking of utilizing that (and setup the weeekly schedule there). What do you think?

Reply:
![image](https://user-images.githubusercontent.com/25760992/221643007-6a19fc2b-2d3f-42d2-b74a-ba137cc3e755.png)

Reply:
You should be able to use cron and a trigger in yaml I think? If so, it is better to define all in one place vs. needing to check multiple places for the schedule.

Reply:
vswhere does not have cron on its compliance yml - I think? I'm indifferent on this one. Could go either way

Reply:
Feel free to close, I'm fairly impartial here. We can also update the schedule in ADO without requiring a PR if we don't use cron here

Reply:
Added on yml